assets/templates/tooltip-component.html

```diff
@@ -123,6 +123,27 @@ <h3>Tooltip</h3>
                 </p>
               </div>
             </div>
+            <h3>Tooltip II</h3>
+            <p>
+              Este tooltip se utliza en el gráfico de Visitas por Dias, semanas
+              y horas de la pagina de reportes.
+            </p>
+            <hr class="dpsg-break" />
+            <div class="wrapper-tooltip">
+              <div class="dp-box">
+                <div class="dp-tooltip">
+                  <p class="wrap-tooltip dpsg-button-example">
+                    Tooltip II example
+                  </p>
+                  <div class="tooltip bgColor">
```

Comment:
only need in webapp this css class? or need all structure?

Reply:
the structure is practically similar, only using the classes should it work